### PR TITLE
Fixes TEG processing not being synchronized.

### LINKED ISSUE #10034: The REAL reason of the TEG fuckery.
#### Description of issue

The TEG's main piece which does all the processing ticks faster than the pipenet, causing it to effectively run at double the speed for the pipenet every few ticks. When this happens it jams because the gas hasn't been moved before last tick.
#### Difference between expected and actual behavior

TEG to tick at the same rate as pipenets, but it actually ticks faster.
#### Steps to reproduce

Make a TEG.
#### Specific information for locating

[`generator.dm`](https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/modules/power/generator.dm)
[`Processes/pipenet.dm`](https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/controllers/Processes/pipenet.dm)
[`Processes/power.dm`](https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/controllers/Processes/power.dm)
#### Length of time in which bug has been known to occur

a6b0c9a818aaabb14abfac4cf5ac50cb19a97c8b
#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [ ] Issue happened in a recent (less than 7 days ago) round (has the TEG been set up in the last 7 days?)
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by PJB3005**
Fixes #10034

Welcome back old TEG.

### CODE REVIEW

File: code/controllers/Processes/pipenet.dm [hunk 1]
```diff
@@ -1,14 +1,16 @@
-var/global/list/datum/pipe_network/pipe_networks = list()
-var/global/list/obj/machinery/atmospherics/atmos_machines = list()
+var/list/datum/pipe_network/pipe_networks = list()
```
Comment (ComicIronic) [inline code/controllers/Processes/pipenet.dm]: But why

Comment (PJB3005) [inline code/controllers/Processes/pipenet.dm]: Because `global` is redundant and it tickled my autism.

Comment (clusterfack) [inline code/controllers/Processes/pipenet.dm]: It's redundant for now but what if lummox makes an extremely unlikely change that makes it necessary?

Comment (PJB3005) [inline code/controllers/Processes/pipenet.dm]: Cluster I know you're meming but strong typing is more likely to happen than this.

**Comment by clusterfack** [comment]
Also this is a retarded way to fix the bug

**Comment by PJB3005** [comment]
How would YOU do it then, clusterfack?

**Comment by AutismSpider** [comment]
Make sure it works you meme loving fuck

File: code/controllers/Processes/pipenet.dm (hunk 1 above)
Comment (ComicIronic) [inline code/controllers/Processes/pipenet.dm]: I prefer you keep global for readability.

Comment (PJB3005) [inline code/controllers/Processes/pipenet.dm]: I don't see why because any bar outside a proc or class is global.

Comment (ComicIronic) [inline code/controllers/Processes/pipenet.dm]: And to anyone that might be confused about that having global there in the var declaration would make it clear.

Absolute pathing is also redundant, but there are steps we take to make code easier to understand.

**Comment by PJB3005** [comment]
@AutismSpider, yes, it works fucking perfectly.

File: code/controllers/Processes/pipenet.dm (hunk 1 above)
Comment (PJB3005) [inline code/controllers/Processes/pipenet.dm]: Sorry but what could a variable OUTSIDE of a proc or class possibly do?

Comment (PJB3005) [inline code/controllers/Processes/pipenet.dm]: Let's settle this matter!

https://www.strawpoll.me/10272898

**Comment by PJB3005** [comment]
@clusterfack still waiting on a better way to fix it if it's so retarded.

File: code/controllers/Processes/pipenet.dm (hunk 1 above)
Comment (Kurfursten) [inline code/controllers/Processes/pipenet.dm]: In matters of purely style, I see no reason to make a fuss if PJB wants code it his way.

**Comment by PJB3005** [comment]
Happy now?

**Pull request merged**